src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.model;
+
+public enum Operation {

```

Comment:
it seems like you already have Operation enum in FruitTransaction, remove redundant